io.github.endless_sky.endless_sky.appdata.xml

```diff
@@ -7,7 +7,7 @@
   <summary xml:lang="de">Weltraumhandels und Kampfsimulator</summary>
   <summary xml:lang="fr">Jeu d'exploration et de combat dans l'espace</summary>
   <developer_name>Michael Zahniser</developer_name>
-  <project_license>GPL-3.0 AND GPL-2.0 AND CC-BY-SA-4.0 AND CC-BY-SA-3.0 AND CC-BY-4.0 AND CC-BY-3.0 AND CC-BY-2.0 AND CC0-1.0</project_license>
+  <project_license>GPL-2.0+</project_license>
```

Comment:
```suggestion
  <project_license>GPL-3.0-or-later</project_license>
```
GPL v2.0 is not allowed.

Reply:
I saw GPL 2.0 in the license and thought that it applied to some parts of the code. I'll update my PR accordingly.

Reply:
The codebase is GPLv3+.
https://github.com/endless-sky/endless-sky/blob/355cdfd83a0d153bdfa9bcd8338047ffd5aeeede/source/main.cpp#L463
The GPLv2 in the copyright file is from a single sound asset.
https://github.com/endless-sky/endless-sky/blob/355cdfd83a0d153bdfa9bcd8338047ffd5aeeede/copyright#L498-L502